### DISCUSSION TITLE
"Share" Functionality for HTML output Template

### DISCUSSION THREAD

**Discussion by paxvas** [microsoft-search/pnp-modern-search#2441] [Q&A]
We would like to include a Share Button for each item in the HTML output which should leverage the ootb functionality from MS Viva Connections Feed Share functionality. This would have the advantage of consistency with other MS products and we do not have to reinvent the wheel, which is already there. Is there a chance that we can utilize it? Any idea whom to ask for code or guidance?

**Comment by wobba** [comment]
Afaik there is no API/available component to third party to invoke the Sharing dialog today. Once this is available we can add support for it.

**Comment by paxvas** [comment]
Thx I thought so. Sharepoint has problem with the wording "share" as it has
multiple meanings ;-)

Mikael Svenson ***@***.***> schrieb am Mi., 12. Okt. 2022,
11:57:
